SmartEVSE-3/src/evse.cpp

```diff
@@ -3383,34 +3672,21 @@ void StartwebServer(void) {
                     ToModemWaitStateTimer = 0;
                     ToModemDoneStateTimer = 0;
                     LeaveModemDoneStateTimer = 0;
-                    setAccess(0);
+                    setAccess(false);
                     break;
                 case 1:
-                    setAccess(!DelayedStartTime.epoch2);              //if DelayedStartTime not zero then we are Delayed Charging
                     setMode(MODE_NORMAL);
                     break;
                 case 2:
-                    if (MainsMeter) {
-                        OverrideCurrent = 0;
-                        setAccess(!DelayedStartTime.epoch2);
-                        setMode(MODE_SOLAR);
-                        break;
-                    }
+                    setMode(MODE_SOLAR);
+                    break;
```

Comment:
These two lines seem incorrect to me: if you have no mainsmeter installed, you can not run in Solar mode!

Reply:
That check is already in `setMode`, and since MQTT does not really reply implicitly when handling the set less code seemed a better outcome to me.

Reply:
Correct, thanks!! 
Like Elon said, Id like to pay my dev's for every line of code they write, but I'd pay them double for every line they remove!